src/lightning_app/components/serve/auto_scaler.py

```diff
@@ -403,7 +403,8 @@ class AutoScaler(LightningFlow):
     Args:
         min_replicas: The number of works to start when app initializes.
         max_replicas: The max number of works to spawn to handle the incoming requests.
-        autoscale_interval: The number of seconds to wait before checking whether to upscale or downscale the works.
+        autoscale_up_interval: The number of seconds to wait before checking whether to upscale the server
+        autoscale_down_interval: The number of seconds to wait before checking whether to downscale the server
```

Comment:
n00b question: Do you think it's reasonable to have the arg names as the following intead? (scale up vs. scale out)
```suggestion
        scale_out_interval: The number of seconds to wait before checking whether to upscale the server
        scale_in_interval: The number of seconds to wait before checking whether to downscale the server
```

Reply:
not noob and great suggestion